src/Copy/BrandCopy.elm

```diff
@@ -118,97 +118,106 @@ brandCopy key =
         PrivacyIntroP ->
             CopyText "This policy relates to www.contemplatingaction.org.uk (“the Site”) which is owned by Neontribe ltd and related activity "
 
-        PrivacyCompanyInfoList ->
+        PrivacyCompanyAddress ->
             CopyList
                 [ "Neontribe ltd trading as Neontribe"
                 , "Registered as a limited company in England & Wales"
                 , "Business registration: 06165574"
                 , "Registered office: 106 Lincoln St. Norwich, Norfolk, NR2 3LB."
                 ]
 
-        PrivacyDataCollectH3 ->
+        PrivacySectionOneH3 ->
             CopyText "What information is collected?"
 
-        PrivacyDataCollectP ->
+        PrivacySectionOneP ->
             CopyText "We collect the following information from people who use this website:"
 
-        PrivacyDataCollectList ->
+        PrivacySectionOneList ->
             CopyList
                 [ "Email addresses of people who communicate with us via email"
                 , "Information volunteered by people in the course of using of the website (predominantly survey responses)"
                 , "Analytics and tracking data about people’s site usage"
                 ]
 
-        PrivacyInfoUsedH3 ->
+        PrivacySectionTwoH3 ->
             CopyText "How is the information used?"
 
-        PrivacyInfoUsedP ->
+        PrivacySectionTwoP ->
             CopyText "Neontribe uses the information you provide to:"
 
-        PrivacyInfoUsedList ->
+        PrivacySectionTwoList ->
             CopyList
                 [ "Improve the usability of the Site"
                 , "Help design future projects "
                 , "Contribute to research about third sector use of digital"
                 , "Send you any information you have requested relating to the Contemplating Action project"
                 ]
 
-        PrivacyInfoSharedH3 ->
+        PrivacySectionThreeH3 ->
             CopyText ""
 
-        PrivacyInfoSharedP1 ->
-            CopyText "If you chose to give us contact details we will only contact you in relation to these four things. You can also tell us if you no longer wish to receive information from Neontribe by replying to any email from us with the subject line Unsubscribe, or sending a message to info@neontribe.co.uk requesting to unsubscribe."
-
-        PrivacyInfoSharedP2 ->
-            CopyText "Information that is identifiable as relating to you (i.e. it has not been edited to make it anonymous) is not shared with or sold to other organizations for commercial purposes, other than under the following circumstances:"
+        PrivacySectionThreeP ->
```

Comment:
There's another issue to work out best method of flexible number of `p`. There are 2 places we need that here. For now, these will render as List Items in contemplating action. I am not sure if Hospice has the same need.

Reply:
Changed my mind. Hardcoded for now. At least in Chrome - nothing appears if you put an empty `<p>`. I am hoping it's elm stripping them out - but not verified that.

Means we can use `CopyText ""` for unused `Keys`.